### DISCUSSION TITLE
shortcut to jump to start of an header, in jupyter notebook

### DISCUSSION THREAD

**Discussion by ggrrll** [microsoft/vscode-discussions#1109] [Extension Development QnA] [answered]
Hi, 

thanks for all the extensions, and in particular the support for jupyter

what I miss is a quick way to collapse the whole md section, once I am in the middle of it:

* on jupyter lab/notebook, you can just press 'left arrow key', to quickly jump to the header

* on VS code , I have to move all the way up to the beginning of the section , which makes me waste quite some time...

is there already an existing shortcut ? how to implement it?

thanks

ps: my specs:

Version: 1.87.2 (Universal)
Commit: 863d2581ecda6849923a2118d93a088b0745d9d6
Date: 2024-03-08T15:21:31.043Z
Electron: 27.3.2
ElectronBuildId: 26836302
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Darwin x64 23.1.0

**Comment by isidorn** [comment, marked as answer]
Sounds like this is a feature request for VS Code notebooks. Thus. closing this, as this community is for vscode extension authors
